arch/arc/core/mpu/arc_mpu_common_internal.h

```diff
@@ -238,7 +238,7 @@ int arc_core_mpu_buffer_validate(const void *addr, size_t size, int write)
  * This function provides the default configuration mechanism for the Memory
  * Protection Unit (MPU).
  */
-static int arc_mpu_init(void)
+int arc_mpu_init(void)
```

Comment:
We don't use return value anywhere.

```suggestion
void arc_mpu_init(void)
```

Reply:
Right, but it does return some error codes, so either we remove those, check for the errors when calling or ignore.

Reply:
addressed